tox.ini

```diff
@@ -1,32 +1,32 @@
 [tox]
-minversion = 3.14.6
+minversion = 3.21.4
 envlist =
-    py{37,38,39,310}-django32-wagtail{41,42,50}-factoryboy32
-    py{38,39,310}-django40-wagtail{41,42,50}-factoryboy32
-    py{38,39,310,311}-django41-wagtail{41,42,50}-factoryboy32
+    py{38,39,310,311}-django42-wagtail{52,60}-factoryboy{32,33}
+    py{310,311,312}-django50-wagtail60-factoryboy{32,33}
     coverage-report
     lint
 
+[gh-actions]
+python =
+    3.8: py38
+    3.9: py39
+    3.10: py310
+    3.11: py311
+    3.12: py312
+
 [testenv]
 commands = coverage run --parallel -m pytest {posargs}
 extras = test
-passenv =
-    TEST_DB_NAME
-    TEST_DB_USER
-    TEST_DB_PASSWORD
-    TEST_DB_HOST
-    TEST_DB_PORT
 deps =
-    django32: django>=3.2,<3.3
-    django40: django>=4.0,<4.1
-    django41: django>=4.1,<5.0
-    wagtail41: wagtail>=4.1,<4.2
-    wagtail42: wagtail>=4.2,<5.0
-    wagtail50: wagtail>=5.0,<5.1
+    django42: django>=4.2,<5.0
+    django50: django>=5.0,<5.1
+    wagtail52: wagtail>=5.2,<6.0
+    wagtail60: wagtail>=6.0,<6.1
     factoryboy32: factory-boy>=3.2,<3.3
+    factoryboy33: factory-boy>=3.3,<3.4
 
 [testenv:coverage-report]
-basepython = python3.7
```

Comment:
go with at least `3.11`. no reason not to use a faster python version

Reply:
Changed in c6dc298